src/main/java/org/sagebionetworks/bridge/sdk/models/subpopulations/Subpopulation.java

```diff
@@ -0,0 +1,161 @@
+package org.sagebionetworks.bridge.sdk.models.subpopulations;
+
+import static org.sagebionetworks.bridge.sdk.utils.Utilities.TO_STRING_STYLE;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.Objects;
+import java.util.Set;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import org.sagebionetworks.bridge.sdk.json.SubpopulationGuidDeserializer;
+import org.sagebionetworks.bridge.sdk.models.holders.GuidVersionHolder;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.common.collect.ImmutableSet;
+
+public final class Subpopulation {
+    
+    private String name;
+    private String description;
+    @JsonDeserialize(using=SubpopulationGuidDeserializer.class)
+    private SubpopulationGuid guid;
+    private boolean required;
+    private boolean defaultGroup;
+    private Long version;
+    private Integer minAppVersion;
+    private Integer maxAppVersion;
+    private Set<String> allOfGroups;
+    private Set<String> noneOfGroups;
+    
+    public Subpopulation() {
+        // This is different than elsewhere in the SDK...
```

Comment:
Could you please expand on this comment with why it is different? Also, sign/date.  Thanks